core/src/test/java/com/graphhopper/reader/dem/TunnelElevationInterpolatorTest.java

```diff
@@ -0,0 +1,306 @@
+package com.graphhopper.reader.dem;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.graphhopper.coll.GHBitSetImpl;
+import com.graphhopper.reader.ReaderWay;
+import com.graphhopper.routing.util.DataFlagEncoder;
+import com.graphhopper.routing.util.EncodingManager;
+import com.graphhopper.storage.GraphExtension;
+import com.graphhopper.storage.GraphHopperStorage;
+import com.graphhopper.storage.NodeAccess;
+import com.graphhopper.storage.RAMDirectory;
+import com.graphhopper.util.EdgeIteratorState;
+
+import gnu.trove.set.hash.TIntHashSet;
+
+public class TunnelElevationInterpolatorTest {
```

Comment:
Do we really need a separate test for tunnels? What is different with them from the code concept? Or maybe create an abstract tester with two subclasses or one test with two instances?